tools/topology/topology2/cavs-sdw.conf

```diff
@@ -11,7 +11,7 @@
 <virtual.conf>
 <host-copier-gain-mixin-playback.conf>
 <mixout-gain-dai-copier-playback.conf>
-<dai-copier-gain-module-copier-capture.conf>
+<dai-copier-eqiir-module-copier-capture.conf>
```

Comment:
Would this be needed for SDW DMICs too ?

Reply:
I'd assume the SDW codec can be set for suitable gain and it hopefully also handles the mic start transient for ALH DAI.

This change is impacting the cAVS/ACE direct attach DMIC DAI usage. It seems to be enabled in many SDW platform topologies. E.g. sof-mtl-rt711-4ch.tplg:

![sof-mtl-rt711-4ch](https://user-images.githubusercontent.com/28754793/220648522-560ac33f-094b-4f80-89db-38fa56e60947.png)

sof-tgl-rt711-rt1308-4ch.tplg:

![sof-tgl-rt711-rt1308-4ch](https://user-images.githubusercontent.com/28754793/220649115-abbcccf8-e84d-4cf8-be6d-2b8835ae9cd9.png)

sof-adl-rt711-4ch.tplg:

![sof-adl-rt711-4ch](https://user-images.githubusercontent.com/28754793/220649514-5be75cc1-8e46-4e5f-94cd-790c375ddb69.png)

Reply:
Can we do direct DMICs first and then see if this is needed for SDW ?

> I'd assume the SDW codec can be set for suitable gain and it hopefully also handles the mic start transient for ALH DAI.

You don't sound sure, probably better to add direct DMIC first and then evaluate SDW DMICs as another PR ?

Reply:
> Can we do direct DMICs first and then see if this is needed for SDW ?
> 
> > I'd assume the SDW codec can be set for suitable gain and it hopefully also handles the mic start transient for ALH DAI.
> 
> You don't sound sure, probably better to add direct DMIC first and then evaluate SDW DMICs as another PR ?

Yes, I prefer other PR. I just got access with help from @keqiaozhang to a suitable device with SDW and I will test with it.